### PR TITLE
Feat-5: "Delete Posts that are 'Bad'"

### PR DESCRIPTION
**Pull request by sxmrxzxr**
The following features have been added:

1. Posts now have a boolean attribute in their schema called 'visible'
2. when a post has 5 or more downvotes, visible is set to false
3. getting posts by location now filters on visibility

### CODE REVIEW

File: routes/posts.js [hunk 1]
```diff
@@ -109,6 +118,9 @@ router.patch('/:id', function(req, res, next) {
 
     Post.findByIdAndUpdate(req.params.id, update, options, function(err, post) {
         if (err) return next(err);
+        if (post.downvotes >= 5) {
+            post.visible = false;
```
Comment (ryfellow) [inline routes/posts.js]: Shouldn't a post's visibility be set based on (upvotes - downvotes) <= -5?

File: routes/posts.js [hunk 2]
```diff
@@ -25,9 +25,18 @@ router.get('/location', function(req, res, next) {
 
     userLocation = [parseFloat(req.query.lng), parseFloat(req.query.lat)];
 
-    var query = {"location.coordinates": {$nearSphere: {$geometry:
-                {type: "Point", coordinates: userLocation},
-                $maxDistance : CONFIG.Posts.defaultDistance}}};
+    var query = {
+                    "visible": {$ne: false},
```
Comment (adeutry) [inline routes/posts.js]: There has to be a better way

File: routes/posts.js [hunk 3]
```diff
@@ -109,6 +118,9 @@ router.patch('/:id', function(req, res, next) {
 
     Post.findByIdAndUpdate(req.params.id, update, options, function(err, post) {
         if (err) return next(err);
+        if (post.downvotes >= 5) {
```
Comment (adeutry) [inline routes/posts.js]: Make this configurable pls

File: routes/posts.js (hunk 1 above)
Comment (adeutry) [reply routes/posts.js]: +1

**Pull request closed without merging**